tests/usb_packet.py

```diff
@@ -363,8 +521,9 @@ def __init__(self, **kwargs):
         self.crc5 = kwargs.pop('crc5', crc5)
 
         # Always override to match IFM
+        # Only required for < XS3?
         #self.data_valid_count = 4 #todo
-        self.data_valid_count = 0
+        #self.data_valid_count = 0
```

Comment:
Can we delete this?

Reply:
Added a TODO comment for XS2 and removed commented out code